.github/workflows/security-action.yaml

```
@@ -0,0 +1,17 @@
+name: Security Action
+on:
+  pull_request: {}
+jobs:
+  scan:
+    env:
+      SEMGREP_APP_TOKEN: ${{ secrets.SEMGREP_APP_TOKEN }}
+      JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+      JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+    runs-on: [feature/generic, size/sm, env/infra-mgmt, group/default]
```

Self hosted runners are not available on public repos
```suggestion
    runs-on: ubuntu-latest
```